src/platforms/customer.io/index.ts

```diff
@@ -0,0 +1,15 @@
+import { client } from '@/platforms/customer.io/client';
+import { icon } from '@/platforms/customer.io/icon';
+import { auth, platform } from '@/sdk';
+
+export default platform('customer.io', {
+  auth: auth.apiToken(),
+  display: {
+    name: 'Customer.io',
+    iconURI: icon,
+    categories: [],
```

Comment:
Let's add this to CRMs?

Reply:
What happens if someone tries to hit the CRM APIs for this?

Also customer.io is more of a hybrid between CDP and marketing automation. Thinking we can leave it out for now?

Reply:
It's more of a hybrid between a cdp and marketing automation. Per [G2](https://www.g2.com/products/customer-io/reviews), I'll add marketing automation for now?